### DISCUSSION TITLE
Create desktop version of the app

### DISCUSSION THREAD

**Discussion by wadbr** [iSoron/uhabits#693] [Feature Requests]
Hello, I love this app but I use my desktop the most when working. Is there any plans or possibility this would be ported to desktop as an electron app?

**Comment by iSoron** [comment]
Thank you for the feature request, @wadbr. We do have plans to make the app multiplatform (see #486). If anyone else is interested in a desktop version of Loop, please leave your comments here or give this thread a thumbs-up, so we know there's user demand for it.

**Reply by asif-haque** [reply]
You could make a web version

**Reply by k4j8** [reply]
If you're looking for a web version, I would check out [Beaver Habit Tracker](https://github.com/daya0576/beaverhabits).

**Reply by aieng-imran** [reply]
I'm very interested for desktop version.

**Reply by juil** [reply]
@iSoron Has there been any progress on a desktop or web version?

**Comment by Warden20** [comment]
This is a great idea

**Reply by srrdasa** [reply]
Yes very much required.

**Reply by bsaunder2002** [reply]
back in 2018, I setup a tcl_myschedule_to_loophabitsdb script written im TCL to take a personal db of my schedule and convert it to the Habit's db format so that it could be imported.
Allowed me to setup my schedule in a database manager gui app on my desktop, run the script, and import it into habits.
I havent touched it since but it wasnt that hard.  Tcl/Tk is super easy with its built in sqlite3 libs.  Python would be fine too.

**Reply by mahdikhashan** [reply]
Hi, I can Help with both the desktop or Webapp version.
Both front and backend. I prefer the front to be built with Tailwind and vue. In the back I'm good at django but kotlin would be better.

**Comment by herrbrixter** [comment]
I'd love to have a Windows version because I'm more productive there. I want to avoid using my mobile Android devices because I waste a lot of time there.
